LivlReviews.Api/DTO/LoginRequest.cs

```diff
@@ -4,8 +4,6 @@ namespace LivlReviews.Api.Models;
 
 public class LoginRequest
 {
-    [Required]
     public string? Email { get; set; }
```

Comment:
why everything is nullable al all your DTO ?

Reply:
Because when you do the request from a client, the client can forget a property, so they may not be here.
Checking if a non-nullable value is null in the controller trigger a warning.

Reply:
C# miss a schema validation system 😒